lib/actions-oclif/key/index.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ * @license
+ * Copyright 2016-2020 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { flags } from '@oclif/command';
+import { stripIndent } from 'common-tags';
+import Command from '../../command';
+import * as cf from '../../utils/common-flags';
+import { getBalenaSdk, getVisuals } from '../../utils/lazy';
+import { parseAsInteger } from '../../utils/oclif-validation';
+
+type IArg<T> = import('@oclif/parser').args.IArg<T>;
+
+interface FlagsDef {
+	help: void;
+}
+
+interface ArgsDef {
+	id: number;
+}
+
+export default class KeyCmd extends Command {
+	public static description = stripIndent`
+		Display an SSH key.
+
+		Show information about a single SSH key.
+	`;
+
+	public static examples = ['$ balena key 17'];
+
+	public static args: Array<IArg<any>> = [
+		{
+			name: 'id',
+			description: 'key id',
```

Comment:
Suggestion: `balenaCloud database ID for the SSH key`
(Because one might wonder if 'key id' means the ssh key fingerprint or something...)

Reply:
Hmm yeah.  Not sure I like `balenaCloud database ID` though - seems like we are exposing internal implementation details a bit?

Reply:
What about `balenaCloud ID for the SSH key`

Reply:
Sounds good 👍 (`balenaCloud ID for the SSH key`)